wideboost/wrappers/wxgb.py

```diff
@@ -106,8 +106,10 @@ def get_eval_metric(params,obj):
         'squarederror':eval(squarederror,obj,'squarederror'),
         'rmse':eval(rmse,obj,'rmse')
     }
-    print("Taking first argument of eval_metric. Multiple evals not supported using xgboost backend.")
-    return output_dict[params['eval_metric'][0]]
+    if isinstance(params['eval_metric'], list):
```

Comment:
Can you change to `isinstance(params['eval_metric'], list) or isinstance(params['eval_metric'], tuple)`?  `hyperopt` submits the parameters as tuples, and, without this change, the examples break.

Also, not needed for this merge, but if you wanted to put in a test that tests this functionality, I'd be very appreciative.

Reply:
Thanks for reviewing the PR! I added another check for tuple, as you suggested. However, I couldn't get the test to pass with wxgb.py functions. This file imports xgboost, which is not installed when running tests.